config/images.yml

```
@@ -156,6 +156,9 @@
     - "media/images/content/hero-images/0025--mobile.jpg"
     - "media/images/content/hero-images/0025--tablet.jpg"
 
+"media/images/content/ttalianne.jpg":
+  alt: "Ttalianne, a teacher training adviser."
```

I'm presuming her name is actually Lianne not Ttalianne?


```suggestion
  alt: "Lianne, an explore teaching adviser"
```

---

I think Steph's off now so have asked Christine to confirm the adviser's name (the file is called Ttalianne)